### ISSUE TITLE
Archive for accounts, tags and categories

### ISSUE THREAD

**Issue by alehano** [mvarnagiris/financius#264]
I would like to hide some of my accounts. For example old credit cards. I dont use it anymore, but if I'll delete it, it's also deletes all it's transaction.

**Comment by mvarnagiris** [comment]
Good point. I need to add archive function for accounts, categories, tags

**Comment by jeavilez** [comment]
Perhaps it would be better to have an option to hide accounts in account panel while they are with zero balance

**Comment by alehano** [comment]
I'm not sure about it. Sometimes I have active zero balanced accounts.
